site/src/components/CopyAndPastedNavBar.md

```diff
@@ -0,0 +1,2 @@
+05/01/2022 - Chris Chan
+We would love to remove NavBar from Kaizen, the NavigationBar and OffCanvas components should be refactored and deal with when possible. It's currently being copied to both /site and /storybook
```

Comment:
Creating these kind of files isn't really a convention that we follow. What I would recommend is putting a comment on the `NavigationBar.tsx` file to explain that it was copy pasted over from an old Kaizen component.

Reply:
Yeah, that make sense, I was trying to get a comment from you 😃